app/views/annotations/_form.html.erb

```diff
@@ -0,0 +1,151 @@
+<div id="annotation-form">
+  <%= form_with model: @annotation do |f| %>
+    <b><%= t('admin.annotation.time') %></b> &nbsp <%= TimeStamp.new(total_seconds: @total_seconds).hms_colon_string %>
+
+    <br><br>
+
+    <!--- HIDDEN FIELDS -->
+    <%= f.hidden_field :medium_id, value: @medium_id %>
+    <%= f.hidden_field :total_seconds, value: @total_seconds %>
+    <div hidden>
+      <div id="annotation_id">
+        <%= @annotation.id %>
+      </div>
+       <!--- for prefilling the edit form -->
+      <div id="annotation_subtext">
+        <%= @annotation.subtext %>
+      </div>
+    </div>
+
+    <!--- COMMENT TEXT AREA -->
+    <b><%= t('admin.annotation.comment') %></b>
+    <%= f.text_area :comment, class: 'form-control' %>
```

Comment:
Ensure no empty comments are stored. See [here](https://github.com/MaMpf-HD/mampf/pull/527/files#diff-40587fc8ef28ff3ce7ef7f060ae9ecd81d7dfa7697b881968c4c54d55844caa9R24-R29). Make sure to also validate in backend, not only frontend.

Reply:
Wouldn't you say that in this case it's a feature to have the possibility of a blank comment field?
Maybe, sometimes users only want a marker on their video timeline without text. But I'm open for other opinions!